docs/learn/learn-staking-faq.md

```diff
@@ -69,8 +48,8 @@ staking reward for the eras when your nomination is applied to that specific val
 ### Nominated an oversubscribed validator
 
 Only the top 256 nominators of a Validator receive the staking rewards. If your nomination is
-applied to an oversubscribed Validator and your stake does not put you in the top 256, you will not
-earn rewards. On a regular basis, ensure that the Validators you chose are not oversubscribed.
+applied to an oversubscribed validator and the size of your stake does not put you in the top 256, you will not
+earn rewards. On a regular basis, ensure that the validators you chose are not oversubscribed or that your stake is bigger than the stake of the last nominator.
```

Comment:
"or that your stake is bigger than the stake of the last nominator." - do you mean that the stake should be bigger than the min-active nomination threshold? It may be better to leave this one out as we have a sub-section for it.

Reply:
Changed as requested.